tests/Application.Tests/Application.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <AssemblyName>Symbolica.Application.Tests</AssemblyName>
     <RootNamespace>Symbolica.Application</RootNamespace>
+    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
Same as the other one of these, I want to kill this. If some hack like this simply had to live in the Deserialization project, which is the immediate consumer of LLVMSharp, then I could probably live with it. But the way I have it at the moment it is the end user which has to remember to do this. Needs to be improved with a better understanding of RIDs and NuGet. I think there is some meta-packaging or something going on here that is not quite working, but I have not gotten to the bottom of it yet.